website/docs/reference/project-configs/test-paths.md

```diff
@@ -13,7 +13,7 @@ test-paths: [directorypath]
 
 ## Definition
 
-Optionally specify a custom list of directories where [singular tests](/docs/build/tests) are located.
+Optionally specify a custom list of directories where [singular tests](/docs/build/data-tests) are located.
```

Comment:
Doesn't this config also specify where your generic data tests are defined?

With the addition of unit tests it will also be the location of your fixtures/ folder.